core/capabilities/ccip/configs/evm/contract_reader.go

```diff
@@ -83,6 +83,10 @@ var DestReaderConfig = evmrelaytypes.ChainReaderConfig{
 					ChainSpecificName: mustGetMethodName("getSourceChainConfig", offrampABI),
 					ReadType:          evmrelaytypes.Method,
 				},
+				"OffRampGetAllSourceChainConfigs": {
```

Comment:
let's add this to consts to maintain the style in the rest of the file

Reply:
```suggestion
				// TODO: replace with const when chainlink-ccip is updated
				"OffRampGetAllSourceChainConfigs": {
```

Reply:
I was tempted to make a chainlink-ccip PR just to add the const, but there are some pending multi-repo merges that make updating the dependency difficult.

Reply:
TODO is fine for now